knockout-pre-rendered.js

```diff
@@ -475,16 +590,23 @@
       var value = valueAccessor();
       
       if (isObjectWithExplicitValues(value)) {
-        setExplicitObjectValues(value, viewModel);
+        setExplicitObjectValues(value, viewModel, allBindings);
       } 
       else if (hasAttributeBinding(allBindings)) {
         initAttributeObservables(element, value, allBindings);
       }
       else {
-        initObservable(element, value, allBindings)  
+        initObservable(element, value, allBindings);
       }
     }
   };
 
+  generatePropertyWritersForBinding("init", "_ko_prerender_initPropertyWriters", "field", function (expression) {
```

Comment:
This also looks like a private field. See my previous question.

Reply:
`_ko_prerender_initPropertyWriters` is a custom name, meant to be internal to *knockout-pre-render*.  Not internal to Knockout itself.

Reply:
Ah okay, thanks for explaining.